src/index.ts

```diff
@@ -282,3 +279,49 @@ export const gt = (versionA: string | null, versionB: string | null): boolean =>
 export const lt = (versionA: string | null, versionB: string | null): boolean => {
 	return compare(versionA, versionB) < 0;
 };
+
+/**
+ * @typedef {Object} SemverObject
+ * @property {string} raw - The original version string
+ * @property {number} major - The major version number
+ * @property {number} minor - The minor version number
+ * @property {number} patch - The patch version number
+ * @property {Array.<string|number>} prerelease - An array of prerelease values
+ * @property {Array.<string|number>} build - An array of build values
+ * @property {string} version - The version containing just major, minor, patch
```

Comment:
Doesn't matter for now, but as food for thought later: we could very easily add interpretation for parts of the standard build metadata we include in here, so we can easily use that information elsewhere. E.g. `.rev`, `.variant` (dev/prod/?).